hardware/vvadd/vvadd_host.cpp

```diff
@@ -64,12 +64,23 @@ int main(int argc, char** argv) {
     }
 
     // setup runtime
-    RunTime rt(
+    //RunTime rt(
+    //     u280::name,
+    //     xclbin_path,
+    //     {"vvadd"}
+    // );
```

Comment:
The commented code no longer works. You should remove them instead of keeping it there. That is not a proper use of comments in deliverable code.

Reply:
Ok. I will delete it.